crates/sui-transactional-test-runner/src/args.rs

```diff
@@ -266,9 +270,9 @@ impl SuiValue {
             None => bail!("INVALID TEST. Unknown object, object({})", fake_id),
         };
         let obj_res = if let Some(v) = version {
-            test_adapter.executor.get_object_by_key(&id, v)
+            sui_types::storage::ObjectStore::get_object_by_key(&*test_adapter.executor, &id, v)
```

Comment:
What does this change do?

Reply:
This helps disambiguate the trait function being called as there are multiple fn defs with same name.